guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -321,6 +346,83 @@ A new Content View called `Import-Library` is created in the target organization
 This Content View is used to facilitate the library content import.
 
 
+
+=== Example Scenarios
+
+The import and export functionality can be used mainly in the following scenarios:
```

Comment:
Lines 352-361 appear to be a repeat of the scenarios described at the beginning of the file.  Do we need to repeat this info?

Also, would it be better move the definition of air-gapped and disconnected scenarios to the section at the beginning of the file/chapter?  The intent would be to describe to the user that this feature has been designed with a goal to support these scenarios.  (That said, the steps in the examples does not need to be in the intro). 

It may also be important to note somewhere (if not already mentioned) that content views associated with imported content view versions, may not be managed on that server (i.e. they are locked).

Reply:
> Also, would it be better move the definition of air-gapped and disconnected scenarios to the section at the beginning of the file/chapter?

This section was called 'Best Practices'  and then got moved to `Example Scenarios`. Hmm the way I see it we first introduce the feature, show things you can do with import/export (how to xyz..) and then say "if you  are in this situation do this."

@melcorr Do you prefer moving the full section to  the top ?

Reply:
More I think of it. I think top down is better. Going to implement that